jobs.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+package de.jobs;
+
+option go_package = "jobs";
+
+option java_package = "org.cyverse.jobs";
+option java_outer_classname = "Jobs";
+
+enum Platform {
+    Condor = 0;
+    AGAVE  = 1;
+}
+
```

Comment:
Some comments at the top of each message definition that briefly describes the purpose of each message might be helpful here.